### PR TITLE
Avoiding to remove info, but the ordering is not preserved.

### PR DESCRIPTION
**Pull request by Joao-L-S-Almeida**

### CODE REVIEW

File: terratorch/cli_tools.py [hunk 1]
```diff
@@ -130,30 +130,31 @@ def write_on_epoch_end(self, trainer, pl_module, predictions, batch_indices):  #
                 save_prediction(prediction, file_name, output_dir, dtype=trainer.out_dtype)
 
 
-def clean_config_for_deployment_and_dump(config: dict[str, Any]):
+def clean_config_for_deployment_and_dump(config: dict[str, Any], clean:bool=False):
```
Comment (CarlosGomes98) [inline terratorch/cli_tools.py:133]: weird spacing in the keyword arguments

**Comment by CarlosGomes98** [comment]
I don't see any way for this `clean` argument to be passed to this function when the user is training, either with a yaml config file or programmatically. This could be passed in the config itself.

**Comment by Joao-L-S-Almeida** [comment]
I transformed it into an input argument: dce51521bf6296b571029225dcf1e9718091ccb7
I believe that's right now.

**Comment by Joao-L-S-Almeida** [comment]
@CarlosGomes98 What will be the destiny for this PR ? Should we close it without merging ?

**Comment by Joao-L-S-Almeida** [comment]
Closing since #109 does it in a better way.

**Pull request closed without merging**